client/tests/integration/permissions.rs

```diff
@@ -315,10 +316,14 @@ fn stored_vs_granted_token_payload() -> Result<()> {
     // Allow alice to mint mouse asset and mint initial value
     let mouse_asset = AssetId::new(asset_definition_id, mouse_id.clone());
     let allow_alice_to_set_key_value_in_mouse_asset = Grant::permission(
-        Permission::from_str_unchecked(
+        Permission::new(
             "CanSetKeyValueInUserAsset".parse().unwrap(),
-            // NOTE: Introduced additional whitespaces in the serialized form
-            &*format!(r###"{{ "asset_id" : "xor#wonderland#{mouse_id}" }}"###),
+            JsonString::from_json_string_unchecked(format!(
+                // Introducing some whitespaces
+                // This way, if the executor compares just JSON strings, this test would fail
+                r##"{{ "asset_id"   :   "xor#wonderland#{}" }}"##,
+                mouse_id
+            )),
```

Comment:
```suggestion
            JsonString::from_json_string_unchecked(format!(
                // Introducing some whitespaces
                // This way, if the executor compares just JSON strings, this test would fail
                r##"{{ "asset_id"   :   "xor#wonderland#{}" }}"##,
                mouse_id
            )),
```

was it not possible to create string with `json!` and then edit it's contents?

Reply:
You mean, how? `json!` creates `serde_json::Value`.